src/pybind/mgr/dashboard/tests/test_osd.py

```diff
@@ -313,7 +335,34 @@ def test_osd_mark_all_actions(self, instance):
         fake_client = mock.Mock()
         instance.return_value = fake_client
         action_list = ['OUT', 'IN', 'DOWN']
+
         for action in action_list:
             data = {'action': action}
             self._task_put('/api/osd/1/mark', data)
             self.assertStatus(200)
+
+        # invalid mark
+        self._task_put('/api/osd/1/mark', {'action': 'OTHER'})
+        self.assertStatus(200)
```

Comment:
```suggestion
        # invalid mark
        instance.reset_mock()
        with self.assertLogs(level='ERROR') as cm:
            self._task_put('/api/osd/1/mark', {'action': 'OTHER'})
            instance.send_command.assert_not_called()
            self.assertIn('Invalid OSD mark action', cm.output[0])
            self.assertStatus(200)
            self.assertJsonBody(None)
```
Make it more meaningful by checking the error log and `CephService.send_command` is not called.